group_5.txt

```diff
@@ -1,6 +1,8 @@
-Phuc Khang:
-Quy Nguyen:
-Vy Tran:
+Phuc Khang: [School], [Fun fact]
+Quy Nguyen: [School], [Fun fact]
+Vy Tran: 
+    - School: Bucknell University
+    - Fun fact: I'm a fast typer
 Linh Nguyen: Tufts , I like making emoji face like this :>> +-+ ://// hehe
```

Comment:
Thanks for formatting your part. Could you also do the same for Linh so everything is consistent